jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -58,17 +58,22 @@ class OutlierDetectionStep(Step):
         snr = string(default='5.0 4.0')
         scale = string(default='1.2 0.7')
         backg = float(default=0.0)
-        save_intermediate_results = boolean(default=False)
+        save_intermediate_results = boolean(default=True)
```

Comment:
Is this param even used any more? It looked to me like all the intermediate files are always saved now.

Reply:
Removed as part of commit 211f90d5.

Reply:
I saw in the diffs where the median and blotted images were being automatically saved to disk, but don't recall seeing if the individually resampled inputs are saved. Are they?

Reply:
This happens in the new code for 'resample.py' starting at line 168 (see the diffs).